app/integration_action.go

```diff
@@ -590,3 +616,38 @@ func (a *App) SubmitInteractiveDialog(request model.SubmitDialogRequest) (*model
 
 	return &response, nil
 }
+
+func (a *App) ExecutePluginAction(request model.PluginIntegrationActionRequest) (*model.PluginIntegrationActionResponse, *model.AppError) {
```

Comment:
First of all, thanks for taking this step :smile: 
That being said, we still need to finish defining what the action should contain. Apart of that, the current design would name this a "wish", and not an "action". Nevertheless, since we are still getting to the details of the design, I am ok leaving it like this for the time being.